src/init.cpp

```diff
@@ -1664,6 +1664,13 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
     if (gArgs.IsArgSet("-seednode")) {
         connOptions.vSeedNodes = gArgs.GetArgs("-seednode");
     }
+    if (gArgs.IsArgSet("-connect")) {
+        // Initiate outbound connections unless connect=0
+        const auto connect = gArgs.GetArgs("-connect");
+        connOptions.bNoConnect = connect.size() == 1 && connect[0] == "0";
```

Comment:
Done (I originally thought it was better to keep the local style and fix them all in batches)